src/app/pages/partners/partner-details/info-request-form/school-selector.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I extracted this from info-request-form back to JS because getting test coverage was a nightmare.